README.md

```diff
@@ -42,7 +42,7 @@ http.createServer((req, res) => {
 
 ### assets.html(opts?)
 Return an `html` stream. Takes the following options:
-- __opts.entry:__ `js` entry point. Defaults to `/bundle.js`
+- __opts.script:__ `js` entry point. Defaults to `/bundle.js`
```

Comment:
Shoot, I forgot to remove this documentation change -- I changed it back to `entry` so as not to pose a breaking change. Anyway, not sure what you mean with your question.